app/models/pet.rb

```diff
@@ -10,4 +14,9 @@ def shelter_name
   def self.adoptable
     where(adoptable: true)
   end
+
+  def self.find_pet(pet)
+    Pet.where("name ILIKE ?", "%#{pet}%")
```

Comment:
It looks like this isn't used for searching for a pet because you used the inherited `search` function (nice work!). So you could take this method out